ledger/src/shred.rs

```diff
@@ -1040,6 +1046,37 @@ pub fn verify_test_data_shred(
     }
 }
 
+#[cfg(feature = "dev-context-only-utils")]
```

Comment:
Can we use an already public `fn make_merkle_shreds_from_entries` with similar semantics instead of exposing this function?

Reply:
I can try. BTW, this is a change related to this prerequisite PR: #6735. I will fix that there as well.